### PR TITLE
Sanitize URL if the integration is ER type

### PR DESCRIPTION
**Pull request by vgarcia13**

### CODE REVIEW

File: cdip_admin/integrations/models/v2/models.py [hunk 1]
```diff
@@ -192,6 +192,18 @@ def _pre_save(self, *args, **kwargs):
             if "broker" not in self.additional:
                 self.additional.update({"broker": "gcp_pubsub"})
 
+        if self.is_er_site:
+            # Cleanup
+            url_parse = urlparse(self.base_url)
+            if url_parse.scheme == "http":
+                url_parse.scheme = "https"
```
Comment (chrisdoehring) [inline cdip_admin/integrations/models/v2/models.py:199]: I'm not confident you'll be able to set this. Can you try this out locally?

Comment (marianobrc) [reply cdip_admin/integrations/models/v2/models.py:199]: agree, please run unit tests locally, and add some new ones to cover this case if you have time

Comment (marianobrc) [reply cdip_admin/integrations/models/v2/models.py:199]: also, can we fix the commits history to include only the commits that are really from this PR?

Comment (vgarcia13) [reply cdip_admin/integrations/models/v2/models.py:199]: I am not sure why all those commits were included, but let me check

**Pull request closed without merging**